recipes/hive_metastore.rb

```diff
@@ -40,6 +40,18 @@
   end
 end
 
+derby_dir =
+  if hive_sql == 'derby'
+    node['hive']['hive_site']['javax.jdo.option.ConnectionURL'][2].split(';').find { |o| /^databaseName/ =~ o }
```

Comment:
``node['hive']['hive_site']['javax.jdo.option.ConnectionURL'][2]`` with default attribute is ``b``.  you sure this works?